packages/rs-drive-abci/tests/strategy_tests/verify_state_transitions.rs

```diff
@@ -59,7 +60,7 @@ pub(crate) fn verify_state_transitions_were_or_were_not_executed(
 
             let consensus_validation_result = match state_transition.transform_into_action(
                 &platform,
-                false,
+                ValidationMode::CheckTx, //using check_tx so we don't validate state
```

Comment:
I would use more abstract name for this mode which is not related to endpoint name (layers separation).

Reply:
This is only testing code, don't really want to make something just for a test.

Reply:
Ok